### PR TITLE
Deployment support

### PR DESCRIPTION
**Pull request by DeathByTape**

### CODE REVIEW

**Comment by jweaver** [comment]
This looks good, but 2 last things we need:

1) We need to create the package in bintray for this project (I'll take care of that).
2) We need the `pom.xml` to have the distribution section for bintray:

```xml
    <distributionManagement>
        <repository>
            <id>bintray-elide-repo</id>
            <url>https://api.bintray.com/maven/yahoo/maven/elide;publish=1</url>
        </repository>
    </distributionManagement>
```

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by DeathByTape** [comment]
@jweaver thanks for reading this over. I updated the `pom.xml` accordingly now

File: .travis.yml [hunk 1]
```diff
@@ -4,5 +4,18 @@ jdk:
 before_install:
   - sudo apt-get install libaio1 # Needed to run mysqld for integration tests
 script: mvn verify
-# TODO: Need to add a deploy line when we have bintray access
-# See: http://docs.travis-ci.com/user/customizing-the-build/
+# TODO: Add coveralls support
+after_success:
+  - test "${TRAVIS_PULL_REQUEST}" == "false" && test "${TRAVIS_TAG}" != "" && mvn deploy --settings travis/settings.xml
+#  - mvn coveralls:report
```
Comment (jweaver) [inline .travis.yml]: You should probably just remove this if you aren't using coveralls.  If you want to use coveralls, there's a bit of setup required for that.

Comment (DeathByTape) [inline .travis.yml]: We plan on adding coveralls support eventually. I'll move the todo line from line 7

**Comment by DeathByTape** [comment]
@jweaver if there is nothing else, just let me know the secrets when you have them. Thanks!

**Comment by jweaver** [comment]
@DeathByTape you bet.  Don't merge this yet, lets wrap up the onboarding-side stuff and then when I'm done creating the keys and encrypting them, we can change the TODOs here and then merge.

**Pull request merged**